content/docs/07-reference/01-ai-sdk-core/02-stream-text.mdx

```diff
@@ -2274,13 +2274,21 @@ To see `streamText` in action, check out [these examples](#examples).
     },
     {
       name: 'consumeStream',
-      type: '() => Promise<void>',
+      type: '(onError?: (error: unknown) => void) => Promise<void>',
```

Comment:
prefer to use optional options object `{ onError? }` to ensure future extensibility